internal/controllers/machinedeployment/machinedeployment_controller_test.go

```diff
@@ -993,3 +995,96 @@ func updateMachineDeployment(ctx context.Context, c client.Client, md *clusterv1
 		return patchHelper.Patch(ctx, md)
 	})
 }
+
+func TestReconciler_reconcileDelete(t *testing.T) {
+	labels := map[string]string{
+		"some": "labelselector",
+	}
+	md := builder.MachineDeployment("default", "md0").WithClusterName("test").Build()
+	md.DeletionTimestamp = ptr.To(metav1.Now())
+	md.Spec.Selector = metav1.LabelSelector{
+		MatchLabels: labels,
+	}
+	mdWithoutFinalizer := md.DeepCopy()
```

Comment:
md also doesn't have a finalizer, so there wouldn't be a difference. Is this intended?

Reply:
fixed, thx